frontends/concrete-python/tests/compilation/test_circuit.py

```diff
@@ -740,3 +740,37 @@ def f(x, y):
     circuit.keys = fhe.Keys.deserialize(keys2)
     result = circuit.decrypt(output)
     assert np.array_equal(result, (sample_x + sample_y) ** 2)
+
+
+def test_simulate_encrypt_run_decrypt(helpers):
+    """
+    Test `simulate_encrypt_run_decrypt` configuration option.
+    """
+
+    def f(x, y):
+        return x + y
+
+    inputset = fhe.inputset(fhe.uint3, fhe.uint3)
+    configuration = helpers.configuration().fork(
+        fhe_execution=False,
+        fhe_simulation=True,
+        simulate_encrypt_run_decrypt=True,
+    )
+
+    compiler = fhe.Compiler(f, {"x": "encrypted", "y": "encrypted"})
+    circuit = compiler.compile(inputset, configuration)
+
+    sample_x, sample_y = 3, 4
+    encrypted_x, encrypted_y = circuit.encrypt(sample_x, sample_y)
+    encrypted_result = circuit.run(encrypted_x, encrypted_y)
+    result = circuit.decrypt(encrypted_result)
+
+    assert result == sample_x + sample_y
+
+    assert hasattr(circuit, "simulator")
```

Comment:
same thing, I would have comments to say that it's to test that things were really not done in FHE

Reply:
Sure.